packages/versioning/src/versioning.ts

```diff
@@ -160,14 +173,16 @@ export function getNameAtVersion(p: Program, t: Type, v: ObjectType): string {
  * @returns version when the given type was added if applicable.
  */
 export function getAddedOn(p: Program, t: Type): Version | undefined {
```

Comment:
I didn't want to move them around because of the effect it would have on the diff, but I can and should definitely create a tracking issue to remove these in March.

Reply:
Sounds good, I would also add `@deprecated` in the jsdoc that will make it show up as deprecated.

Reply:
Updated.